client/src/components/Settings-DisplayNames.vue

```diff
@@ -12,19 +18,21 @@ export default {
   props: ['value'],
   data () {
     return {
-      names: [],
-      edit: false
+      names: []
```

Comment:
Can you put a comment on what the intended structure of names should be? It appears that it's an array of objects, when intuitively it should be an array of strings